### PR TITLE
[stdlib] Implement a padded p2p txn for testing.

### PR DESCRIPTION
**Pull request by runtian-zhou**
## Motivation

As requested by @andll, we want to test out the overall performance when having a larger transaction. This helper function will pad the normal p2p transaction with a bunch of useless returns to get a larger transaction

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Added a test case to make sure this padding behaves the same as a normal p2p transaction.

### CODE REVIEW

File: language/vm/vm_genesis/src/lib.rs [hunk 1]
```diff
@@ -206,6 +206,43 @@ pub fn encode_transfer_script(recipient: &AccountAddress, amount: u64) -> Script
     )
 }
 
+/// Encode a program transferring `amount` coins from `sender` to `recipient` but padd the output
+/// bytecode with unreachable instructions.
+#[cfg(any(test, feature = "testing"))]
+pub fn encode_transfer_script_with_padding(
```
Comment (sblackshear) [inline language/vm/vm_genesis/src/lib.rs]: Why does this code belong in the same file as the genesis generation code? It seems unrelated.

Comment (runtian-zhou) [reply language/vm/vm_genesis/src/lib.rs]: It was where all the other transactions were. Lemme clean this up. (This should also help us clean our dependency story to make client get rid of the VM btw)
